### PR TITLE
PDP-1185: Final clean-up

### PR DESCRIPTION
**Pull request by danibishop**
* normalised all params to start/endtime instead of using cutoff
* removed useless `dict` casts
* typos
* explicit renaming for easier reading
* some extra refactors trying to uniform everything

### CODE REVIEW

File: lib/debt_to_income_ratio_latest/main.py [hunk 1]
```diff
@@ -134,12 +139,15 @@ async def get_debt_to_income_ratio_latest(
     client = AsyncClient(token)
 
     now = datetime(2021, 9, 1)
+    cutoff_interval = timedelta(days=30)
```
Comment (ricpie) [inline lib/debt_to_income_ratio_latest/main.py:142]: Still need to make a case to return None if sum_of_depository_credit_transactions = 0 on line 130?

**Review by ricpie — CHANGES_REQUESTED** [review]
All looks good, just had a comment about variable names, and I think the debt_to_income_ratio_latest feature needs that patch still!

File: lib/average_end_of_day_depository_balance/main.py [hunk 2]
```diff
@@ -57,14 +57,14 @@ async def get_average_end_of_day_depository_balance(
             )
         )
 
-    r = await asyncio.gather(*inst_coroutines)
+    balances_per_institution = await asyncio.gather(*inst_coroutines)
```
[the remark this answers is not in the document]
Comment (danibishop) [reply lib/average_end_of_day_depository_balance/main.py:60]: I went back and forth between both names. I will take the native speaker advice and rename all of these.

File: lib/debt_to_income_ratio_latest/main.py (hunk 1 above)
Comment (danibishop) [reply lib/debt_to_income_ratio_latest/main.py:142]: That has been addressed at #63 !

**Review by benfasoli — APPROVED** [review]
LGTM 🚀 Thanks for the cleanup @danibishop!

**Pull request merged**